src/components/tab-bar/tab-bar.scss

```
@@ -1,21 +1,19 @@
+@forward './partial-styles/tab-bar-scroller.scss';
```

Stylelint (17.3.0)

[error] Unexpected extension ".scss" in `@forward` (scss/load-partial-extension)

(scss/load-partial-extension)